webplugin/js/app/kommunicateCommons.js

```diff
@@ -127,4 +127,24 @@ function KommunicateCommons() {
         temporalDivElement.innerHTML = html;
         return temporalDivElement.textContent || temporalDivElement.innerText || "";
     }  
+    _this.isConversationClosedByBot = function(){
+        var filtered = CURRENT_GROUP_DATA.groupMembers.filter(function(member){
+            return member.userId == CURRENT_GROUP_DATA.lastMessagingMember
+        });
+        return filtered[0] && filtered[0].role == 2
+
+    }  
+
+    _this.getRatingSmilies = function(rating){
+        switch (rating) {
```

Comment:
@akshatdhabi replace it with a json so that switch is not required.

Reply:
@devashishmamgain  done